addons/project/views/project_views.xml

```diff
@@ -462,17 +462,13 @@
                     </header>
                 <sheet string="Project">
                     <div class="oe_button_box" name="button_box" groups="base.group_user">
-                        <button class="oe_stat_button" name="project_update_all_action" type="object" groups="project.group_project_manager">
-                            <div class="ps-4">
-                                <field name="last_update_color" invisible="1"/>
-                                <field name="last_update_status" readonly="1" widget="status_with_color" options="{'color_field': 'last_update_color'}"/>
-                            </div>
+                        <button class="oe_stat_button ps-2" name="project_update_all_action" type="object" groups="project.group_project_manager">
+                            <field name="last_update_color" invisible="1"/>
+                            <field name="last_update_status" readonly="1" widget="status_with_color" options="{'color_field': 'last_update_color'}"/>
                         </button>
-                        <button class="oe_stat_button o_project_not_clickable" disabled="disabled" groups="!project.group_project_manager">
-                            <div class="ps-4">
-                                <field name="last_update_color" invisible="1"/>
-                                <field name="last_update_status" readonly="1" widget="status_with_color" options="{'color_field': 'last_update_color'}"/>
-                            </div>
+                        <button class="oe_stat_button o_project_not_clickable ps-2" disabled="disabled" groups="!project.group_project_manager">
+                            <field name="last_update_color" invisible="1"/>
+                            <field name="last_update_status" readonly="1" widget="status_with_color" options="{'color_field': 'last_update_color'}"/>
```

Comment:
This changes could breaks some custom xpath.

I think we could do something like that instead:
```diff
diff --git a/addons/project/views/project_views.xml b/addons/project/views/project_views.xml
index adf0213be8af..677399d4f500 100644
--- a/addons/project/views/project_views.xml
+++ b/addons/project/views/project_views.xml
@@ -463,13 +463,13 @@
                 <sheet string="Project">
                     <div class="oe_button_box" name="button_box" groups="base.group_user">
                         <button class="oe_stat_button" name="project_update_all_action" type="object" groups="project.group_project_manager">
-                            <div class="ps-4">
+                            <div class="w-100">
                                 <field name="last_update_color" invisible="1"/>
                                 <field name="last_update_status" readonly="1" widget="status_with_color" options="{'color_field': 'last_update_color'}"/>
                             </div>
                         </button>
                         <button class="oe_stat_button o_project_not_clickable" disabled="disabled" groups="!project.group_project_manager">
-                            <div class="ps-4">
+                            <div class="w-100">
                                 <field name="last_update_color" invisible="1"/>
                                 <field name="last_update_status" readonly="1" widget="status_with_color" options="{'color_field': 'last_update_color'}"/>
                             </div>
```